content/docs/guides/_index.md

```diff
@@ -73,36 +82,37 @@ message HelloReply {
 }
 ```
 
-gRPC also uses `protoc` with a special gRPC plugin to
-generate code from your proto file. However, with the gRPC plugin, you get
+gRPC uses `protoc` with a special gRPC plugin to
+generate code from your proto file: you get
 generated gRPC client and server code, as well as the regular protocol buffer
-code for populating, serializing, and retrieving your message types. We'll
-look at this example in more detail below.
-
-You can find out lots more about protocol buffers in the [Protocol Buffers
-documentation](https://developers.google.com/protocol-buffers/docs/overview),
-and find out how to get and install `protoc` with gRPC plugins in your chosen
-language's Quickstart.
+code for populating, serializing, and retrieving your message types. You'll
+see an example of this below.
 
+To learn more about protocol buffers, including how to install `protoc` with the
+gRPC plugin in your chosen language, see the [protocol buffers documentation][PB docs].
 
 #### Protocol buffer versions
-While protocol buffers have been available for open source users for some
-time, our examples use a new flavor of protocol buffers called proto3, which
-has a slightly simplified syntax, some useful new features, and supports
-lots more languages. This is currently available in Java, C++, Python,
+
+While protocol buffers have been available to open source users for some time,
+most examples from this site use protocol buffers version 3 (proto3), which has
+a slightly simplified syntax, some useful new features, and supports more
+languages. Proto3 is currently available in Java, C++, Dart, Python,
 Objective-C, C#, a lite-runtime (Android Java), Ruby, and JavaScript from the
-[protocol buffers GitHub repo](https://github.com/google/protobuf/releases),
-as well as a Go language generator from the [golang/protobuf GitHub
-repo](https://github.com/golang/protobuf), with more languages
-in development. You can find out more in the [proto3 language
-guide](https://developers.google.com/protocol-buffers/docs/proto3) and the
-[reference documentation](https://developers.google.com/protocol-buffers/docs/reference/overview)
-available for each language. The reference documentation also includes a
-[formal specification](https://developers.google.com/protocol-buffers/docs/reference/proto3-spec)
-for the `.proto` file format.
+[protocol buffers GitHub repo][], as well as a Go language generator from the
```

Comment:
FYI you can omit the `[]` at the end of these links. In this case, `[protocol buffers GitHub repo]` would suffice.

Reply:
By suffixing with `[]`, some IDEs (like VS Code) offer edit-time validation of links, which is quite useful! :)

Reply:
In that case, do you mind keeping the `[]` suffix?

Reply:
Good to know! In that case, feel free to disregard. Just a pointer, not a blocker 👍